docs/developer/reproducibility.md

```diff
@@ -28,6 +28,7 @@ This means that rebuilding the image without updating our Dockerfile will
 Here are the necessary variables that make up our image in the `Dockerfile.env`
 file:
 * `DEBIAN_IMAGE_DATE`: The date that the Debian container image was released
+* `DEBIAN_IMAGE_DIGEST`: The date that the Debian container image was released
```

Comment:
I think you meant to write something like this, right?

```suggestion
* `DEBIAN_IMAGE_DIGEST`: The SHA-256 checksum of the Debian container image
```

Reply:
heh, yep! good ol' copypasta. this is resolved in the latest push.